### PR TITLE
Post

### PR DESCRIPTION
**Pull request by kalex19**
Minor refactoring changes.

### CODE REVIEW

**Review by raechelo — CHANGES_REQUESTED** [review]
thanks for cleaning up the colors/background images we decided against! i'm having some trouble with following the code though in the components/containers JS files. I think it's hard to follow tags and possible syntax errors so I'm not sure it's how we should structure the JSX in our files

**Review by saadricklamar — CHANGES_REQUESTED** [review]
Hey Katie,

Nice job! I think some of the re-factoring here makes the syntax difficult to read and follow. Splitting things on to multiple lines when they can fit on one line diverges from the Turing style guide. I'm worried that Leta and Christie would dock us for that. 

Let me know if you have any questions or disagree.
